### PR TITLE
Finalize list of tags

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: anatomy/tracks/config-json.md [hunk 1]
```diff
@@ -175,6 +169,46 @@ Tags are specified in the top-level `tags` field which is defined as an array of
...
+- `execution_mode/interpreted`: code is interpreted directly
+
+## Platform
+
+- `platform/windows`: runs on Windows
+- `platform/mac`: runs on Mac
+- `platform/linux`: runs on Linux
+- `platform/ios`: runs on iOS
+- `platform/android`: runs on Android
+- `platform/web`: runs on Web
```
Comment (SaschaMann) [inline anatomy/tracks/config-json.md]: What does `runs on Web` mean? Runs in a browser?

File: anatomy/tracks/config-json.md [hunk 2]
```diff
@@ -175,6 +169,46 @@ Tags are specified in the top-level `tags` field which is defined as an array of
...
+
+## Runtime
+
+- `runtime/standalone_executable`: runs as standalone executable
+- `runtime/language_specific`: runs on language-specific runtime
+- `runtime/clr`: runs on Common Language Runtime (.NET)
+- `runtime/jvm`: runs on JVM (Java)
+- `runtime/beam`: runs on BEAM (Erlang)
+
+## Used for
```
Comment (SaschaMann) [inline anatomy/tracks/config-json.md:200]: Should these be things that _most_ people use it for or that it _can_ be used for?

Comment (ErikSchierboom) [reply anatomy/tracks/config-json.md:200]: That's a bit of a judgment call, but I would lean more towards the former than the latter. Virtually all programming languages can be used for scientific computations, but few are really built for that purpose.

File: anatomy/tracks/config-json.md (hunk 1 above)
Comment (ErikSchierboom) [reply anatomy/tracks/config-json.md]: It does. Would you prefer `platform/browser`?

**Comment by ErikSchierboom** [comment]
CC @SleeplessByte

File: anatomy/tracks/config-json.md (hunk 1 above)
Comment (SaschaMann) [reply anatomy/tracks/config-json.md]: I think so. Or just add it to the text behind, the `platform/web` tag is fine 🤷

Comment (SleeplessByte) [reply anatomy/tracks/config-json.md]: Platform web runs on browser is fine I think

File: anatomy/tracks/config-json.md (hunk 2 above)
Comment (SleeplessByte) [reply anatomy/tracks/config-json.md:200]: Yeah think of it as "I'd like to do machine learning, what language should i pick".

File: anatomy/tracks/config-json.md (hunk 1 above)
Comment (ErikSchierboom) [reply anatomy/tracks/config-json.md]: Updated!

File: anatomy/tracks/config-json.md [hunk 3]
```diff
@@ -152,15 +152,12 @@ Example:
 
 ## Tags
 
-Tracks can be annotated with tags, which allows searching for tracks with a certain tag combination.
+Tracks can be annotated with tags, which allows searching for tracks with a certain tag combination. 
 
-Tags are specified in the top-level `tags` field which is defined as an array of strings. The following tags can be used (grouped by category):
+A track should choose their tags based on the general usage of their language. For example, imagine a student thinking: "I'd like to do machine learning, what language should I pick?", or "I'd like to learn functional programming, which language should I choose?". If your language would be a good candidate, give it that tag. If your language supports some functional ideas but they're rarely used, or a few people do Machine Learning in it, but its rare, then do not apply those tags.
```
Comment (iHiD) [inline anatomy/tracks/config-json.md]: @ErikSchierboom @SaschaMann @SleeplessByte I added this bit after you reviewed, based on your conversation ^^ (I couldn't post it as a suggestion as this PR didn't previously touch this LOC)

Comment (SaschaMann) [inline anatomy/tracks/config-json.md]:
```suggestion
A track should choose their tags based on the general usage of their language. For example, imagine a student thinking: "I'd like to do machine learning, what language should I pick?", or "I'd like to learn functional programming, which language should I choose?". If your language would be a good candidate, give it that tag. If your language supports some functional ideas but they're rarely used, or a few people do Machine Learning in it, but it's rare, then do not apply those tags.
```

Comment (ErikSchierboom) [reply anatomy/tracks/config-json.md]: Sounds good to me

Comment (SleeplessByte) [reply anatomy/tracks/config-json.md]: Haha yes. Nice.

**Review by SaschaMann — APPROVED** [review]

**Review by iHiD — APPROVED** [review]

**Comment by iHiD** [comment]
In situ 🙂 

Screenshot 2021-01-19 at 21 52 29

**Pull request merged**